### PR TITLE
Remove various deprecated code for Oxygen release

### PR DESCRIPTION
**Pull request by rallytime**
### What does this PR do?
Removes a couple of deprecations that were marked for removal in the Oxygen release.

### CODE REVIEW

**Review by cachedout — CHANGES_REQUESTED** [review]
The boto tests need to account for this deprecation before we can merge this.

**Comment by rallytime** [comment]
When trying to debug these tests, I ran into a different error. I think this has been fixed in earlier branches. I am waiting on a merge forward before I rebase this and start debugging again.

**Comment by rallytime** [comment]
Hm. I can't reproduce these test failures. Something isn't right here. I'll close this and come back to it later.

**Pull request closed without merging**
